web/packages/dmt-core/src/utils/truncatePathString.ts

```diff
@@ -17,10 +17,10 @@ export function truncatePathString(
   let counter: number = 0
   while (truncatedPath.length <= maxLength) {
     previousPath = truncatedPath
-    truncatedPath = truncatedPath.replace(
-      '...',
-      `${splitTextOnFolders[counter]}/...`
-    )
+    //truncatedPath = truncatedPath.replace(
+    //  '...',
+    //  `${splitTextOnFolders[counter]}/...`
+    // )
```

Comment:
should this be added back? or was there any issues?

Reply:
This was wrong, and are now added again.